data/schema/core/actionwml.cfg

```diff
@@ -865,6 +865,15 @@
 		max=infinite
 		{COLOR_KEYS s_int}
 	[/tag]
+	[tag]
+		name="screen_fade"
+		max=infinite
+		{REQUIRED_KEY red s_int}
+		{REQUIRED_KEY green s_int}
+		{REQUIRED_KEY blue s_int}
+		{REQUIRED_KEY alpha s_int}
```

Comment:
I'd suggest making these keys optional (probably defaulting to 0, except for alpha which should maybe default to opaque, though I'm not quite sure what it even does here so I could be wrong).

Reply:
For alpha, 255 fully applies the fade, so the screen will be that colour, and 0 full unapplies the fade, so the screen will be normal. Essentially it's the opacity of the fade surface that covers the display. I'm not sure which would make more sense as a default.

For the colours a 0 default would make sense.